source/extensions/filters/network/common/redis/redis_command_stats.cc

```diff
@@ -15,7 +15,7 @@ RedisCommandStats::RedisCommandStats(Stats::SymbolTable& symbol_table, const std
       prefix_(stat_name_set_->add(prefix)),
       upstream_rq_time_(stat_name_set_->add("upstream_rq_time")),
       latency_(stat_name_set_->add("latency")), total_(stat_name_set_->add("total")),
-      success_(stat_name_set_->add("success")), error_(stat_name_set_->add("error")),
+      success_(stat_name_set_->add("success")), failure_(stat_name_set_->add("failure")),
```

Comment:
Does this need a doc update? Or does this now match the docs?

Reply:
Good catch; doc has been updated